### ISSUE TITLE
Output gives a nan while training

### ISSUE THREAD

**Issue by reemaRaven** [sanghapriya/not_a_banana#1]
Hi Sangho,

Good write up. I’m trying to run this code, below is the output :

> E:\Work\Images for training>python train_the_snowys.py — snowy_9.jpg
> *

*Output :**

> C:\Users\admin\AppData\Local\Programs\Python\Python36\lib\site-packages\h5py\__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` t
> be treated as `np.float64 == np.dtype(float).type`.
>  from ._conv import register_converters as _register_converters
> WARNING:tensorflow:From train_the_snowys.py:70: softmax_cross_entropy_with_logits (from tensorflow.python.ops.nn_ops) is deprecated and will be removed in a future versio
> Instructions for updating:
> Future major versions of TensorFlow will allow gradients to flow
> into the labels input on backprop by default.
> See @{tf.nn.softmax_cross_entropy_with_logits_v2}.
> step 0, training accuracy nan
> step 10, training accuracy nan
> step 20, training accuracy nan
> step 30, training accuracy nan
> step 40, training accuracy nan
> step 50, training accuracy nan
> step 60, training accuracy nan
> step 70, training accuracy nan

Basically this is my directory structure : E:\Work\Images for training>train_the_snowys.py, read_image.py.

Images to be trained are kept here E:\Work\Images for training\snowy.
Why does running your script ask for a parameter as jpg file?

> python train_the_snowys.py — snowy_9.jpg

shouldn't it ask for the directory which goes into as parameter to the function (in **read_image** file) : def prepare_data(img_dir)

In the **training** file also, its expecting a directory :

> file_list, y_image_label = prepare_data(FLAGS.image_dir)

In case you are sending it as a parameter to the function somewhere within the file, could you direct me to do the same?

Much thanks,
Reema
